packages/core/src/components/selection-group/selection-group.stories.js

```diff
@@ -202,3 +204,22 @@ export const WithParent = () => {
 }
 
 WithParent.storyName = 'With a parent';
+
+export const WithHelperText = () => `
+  <fieldset class="hds-selection-group">
+    <legend class="hds-selection-group__legend">Label</legend>
+    <div class="hds-selection-group__items">
+      ${getCheckboxes('Default')}
+    </div>
+    ${getHelperText()}
```

Comment:
We also need to ask accessibility experts how we connect the helper-text into the selection-group.